pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// MkdirAll implementation that is volume path aware for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	if re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`); re.MatchString(path) {
+		return nil
+	}
+
+	// The rest of this method is copied from os.MkdirAll and should be kept
+	// as-is to ensure compatibility.
```

Comment:
Unfortunately, this wouldn't work; MkDirAll is recursive (note the call on line 46).  The function calls itself on each part of the path.  So if we start from "C:\my\new\folder" it will check if that path exists, if not it will call itself with "C:\my\new" and see if that path exists, then "C:\my" and then "C:". Either the folder existing or hitting the root is the terminating case for the recursion. But since it doesn't recognize volume paths, it would trim down to "\" thinking that is the root, then try to create what it saw as the next folder and fail when trying to create "\?" inside "\".  This is a bug in os.MkdirAll, and ideally would be fixed there.  But since we couldn't wait for the language update to get our Windows daemon up and running, this is the right work-around.  Once the behavior of os.MkdirAll is updated, we can revert to using that directly instead of needing this wrapper.